### DISCUSSION TITLE
iCub robot support steel frame structure

### DISCUSSION THREAD

**Discussion by jimingda** [robotology/community#691] [Q&A] [answered]
Hi,  

Where can I find the document detailing the iCub supporting the steel frame structure mentioned below? We aim to replicate a similar setup to support bipedal walking experiments and prevent the robot from falling.

Thanks.
![2024-03-14_14-14_1](https://github.com/robotology/community/assets/7019692/a3fab3a8-6895-499a-966a-698c2ad34490)

**Comment by maggia80** [comment, marked as answer]
hi @jimingda, we are using products from Molift like: https://www.etac.com/products/patient-handling/mobile-hoists/molift-smart-150-0e0e3b17/

but any other lifter is good

**Reply by jimingda** [reply]
Thanks @maggia80 

I don't have any concept of other frames, which lifter do you recommend?

**Reply by maggia80** [reply]
Try to use Portable patient lifter  as keywords

**Reply by jimingda** [reply]
Thanks @maggia80 

Do you know the details about the frame below? Can i replicate a similar one.
 
![2024-03-20_16-34](https://github.com/robotology/community/assets/7019692/7c7e4d9b-6564-42af-be2e-44e5f9b9c63e)

**Comment by Elio75** [comment]
> 
> Do you know the details about the frame below? Can i replicate a similar one.
> 
![image](https://github.com/robotology/community/assets/53298662/01cc46ec-da17-42e5-8906-8735e1906018)


Hi @jimingda 
For the trolley you are indicating we have no information.  For iCub 2.7 we use this stand here, which are profiles similar to those produced by BOSCH. In theory you would only need the base with the wheels because you should have the pole.

![image](https://github.com/robotology/community/assets/53298662/ac3e748a-a7a1-464f-b947-e979c4efe202)

I attach the PDF with the manufacturer's BOM ([Bonechi](https://www.bonechi-ar.it/))
[rc_iit_090_a_020_r_A1.pdf](https://github.com/robotology/community/files/14703326/rc_iit_090_a_020_r_A1.pdf)


CC @maggia80 @Fabrizio69
